src/universal/routes/admin.js

```diff
@@ -0,0 +1,7 @@
+export default {
+  path: 'admin/impersonate/:newUserId',
```

Comment:
I may also move graphiql under /admin in this PR at the end, just to tidy things up a bit.